gensim/models/tfidfmodel.py

```diff
@@ -274,20 +299,24 @@ def __init__(self, corpus=None, id2word=None, dictionary=None, wlocal=utils.iden
             for example 'ntc', 'bpn' and so on, where the letters represents the term weighting of the document vector.
 
             Term frequency weighing:
-                * `n` - natural,
-                * `l` - logarithm,
+                * `b` - binary,
```

Comment:
Do we really need to duplicate SMART documentation here?

resolve_weights is a public function, so it will be visible to everyone via the docs. Would it be sufficient to hyperlink to that function instead?

> For valid values, see the :func:`~gensim.models.tfidfmodel.resolve_weights` function.

Reply:
I am in favor of keeping the duplication, because the SMART letters are standard (we will not be adding new letters any time soon), and removing them from the `TfidfModel` class docstring forces the reader to jump through hoops to find the relevant documentation. Therefore, it is only a minor improvement for the maintainer and a net negative for the reader.

Reply:
I am in favor of adding a `See Also` docstring to `resolve_weights` and `TfIdfTransformer.__init__` to make the maintainer's life easier.

Reply:
Alright, fair enough.